waku/v2/protocol/waku_rln_relay/waku_rln_relay_types.nim

```diff
@@ -98,6 +97,9 @@ const
   ETH_CLIENT* = "ws://localhost:8540/"
 
 const
+  # The relative folder where the circuit, proving and verification key for RLN can be found
+  # Note that resources has to be compiled with respect to the above MERKLE_TREE_DEPTH
+  RLN_RESOURCE_FOLDER* = "vendor/zerokit/rln/resources/tree_height_" & $MERKLE_TREE_DEPTH & "/"
```

Comment:
How come we need this now?

Reply:
In order to not have duplicates of the circuit in nwaku, I thought that it is better to point to zerokit resource folder directly. Since nwaku is calling zerokit API from a different path than the latter, the resource folders should be passed to correctly locate the circuit (i.e., this string cannot be hardcoded in zerokit)

Reply:
Hm, I don't know how exactly but I suspect this might fail when we do distribution. In general vendoring and static resources should be kept separate. Perhaps OK for initial integration where we compile stuff ourselves.

cc @jm-clius